web/src/components/onboarding-modal/onboarding-modal.tsx

```diff
@@ -7,37 +7,50 @@ import URLS from '../../urls';
 import { LocaleLink } from '../locale-helpers';
 import './onboarding-modal.css';
 
-const STEPS: Array<any> = [
+interface Step {
+  title: string;
+  img: string;
+  alt: string;
+}
+interface Props {
+  onRequestClose: () => void;
+}
+
+type StepType = Step;
+
+const STEPS: Array<StepType> = [
   {
     title: 'Track your team and indvidual progress on the challenge dashboard',
-    img: './images/dashboard.png',
+    img: require('./images/dashboard.png'),
     alt: 'dashboard',
   },
   {
     title: 'Invite friends and colleagues to join your challenge team',
-    img: './images/invite.jpg',
+    img: require('./images/invite.jpg'),
     alt: 'invite',
   },
   {
     title:
       'Earn points by contributing your voice and validating recorded clips',
-    img: './images/contribute.png',
+    img: require('./images/contribute.png'),
     alt: 'contribute',
   },
   {
     title: 'Ready to get started?',
+    img: '',
+    alt: '',
```

Comment:
Typescript allows optional types like so:

```
interface Step {
  title: string;
  img?: string;
  alt?: string;
}
```

However, I think the type definition I gave in my previous comment was even more descriptive:

```
interface StepTypeWithoutImage {
  title: string;
}
interface StepTypeWithImage extends StepTypeWithoutImage {
  img: string;
  alt: string;
}
type StepType = StepTypeWithoutImage | StepTypeWithImage;
```

So, I think we can do without the blank `img` and `alt` fields here.

Reply:
@rileyjshaw Yep, I have tried this way. But in the below code. But when I use `data.img` or `data.alt` it will throw the error `Property 'img' does not exist on type 'StepType'.
 Property 'img' does not exist on type 'StepTypeWithoutImage'`